config/management/src/lib.rs

```diff
@@ -151,6 +185,42 @@ pub mod tests {
         assert_eq!(output, 1); // 0 None results in 1 split
     }
 
+    #[test]
+    #[should_panic]
+    #[ignore]
+    fn test_owner_key_invalid() {
+        let namespace = "owner_key";
+        let mut storage = secure_storage(default_secure_backend(namespace.into()));
+        storage.reset_and_clear().unwrap();
+        owner_key(namespace);
+    }
+
+    #[test]
+    #[ignore]
+    fn test_owner_key_valid() {
+        let namespace = "owner_key";
+        initialize_storage(default_secure_backend(namespace.into()));
+        owner_key(namespace);
```

Comment:
should we not be comparing the return value of owner_key(namespace), against the secure storage backend to make sure the returned key is correct? e.g., something like:

assert_eq!(
    storage.get_public_key(constants::OWNER_KEY).unwrap().public_key, 
    owner_key(namespace));

Reply:
Well this would ensure that I'm not comparing owner to operator key, so that's fair.